components/atoms/ToggleGroupItem/toggle-group-item.tsx

```diff
@@ -0,0 +1,26 @@
+import * as GroupToggle from "@radix-ui/react-toggle-group";
+import clsx from "clsx";
+
+interface ToggleGroupItemProps {
+    checked: boolean,
```

Comment:
nit: semi-colons instead of commas for interfaces

```suggestion
    checked: boolean;
```